examples/tts/conf/fastpitch_speaker_adaptation.yaml

```diff
@@ -0,0 +1,307 @@
+# This config contains the default values for training FastPitch speaker adaptation
+# If you want to train model on other dataset, you can change config values according to your dataset.
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+sup_data_path: ???
+sup_data_types: [ "align_prior_matrix", "pitch" ]
```

Comment:
Does this need SpeakerEmbedding and ReferenceAudio as a sup data type?

Reply:
@hsiehjackson let's add reference audio and speaker embedding as ssup data type to the config.